project/build.properties

```
@@ -1 +1 @@
-sbt.version=1.7.2
+sbt.version=1.9.6
```

The latest version is a bit more recent if you want it...

```suggestion
sbt.version=1.10.3
```